src/tools/run-make-support/src/lib.rs

```diff
@@ -27,16 +32,19 @@ pub mod rfs {
 }
 
 // Re-exports of third-party library crates.
+#[rustfmt::skip]
 pub use bstr;
 pub use gimli;
 pub use object;
 pub use regex;
 pub use wasmparser;
 
 // Re-exports of external dependencies.
+#[rustfmt::skip]
 pub use external_deps::{c_build, cc, clang, htmldocck, llvm, python, rustc, rustdoc};
 
 // These rely on external dependencies.
+#[rustfmt::skip]
 pub use c_build::build_native_static_lib;
 pub use cc::{cc, extra_c_flags, extra_cxx_flags, Cc};
 pub use clang::{clang, Clang};
```

Comment:
There are more comments and blank lines below here. All of the comments apply to single lines, so no comments will end up on incorrect places. But ordering will be changed and blank lines removed. Given that this file mostly contains `use` declaration, it might just be better to add a single `#![rustfmt::skip]` at the top to inhibit formatting of the entire file. This was done for a few files in #126776.